### ISSUE TITLE
Document 2.1 UI plugin changes

### ISSUE THREAD

**Issue by bernd** [Graylog2/documentation#145]
In https://github.com/Graylog2/graylog2-server/pull/2564 we fixed an issue with the app prefix and plugins.

UI plugins now need to have the following at the very top of their `src/web/index.jsx` file:

``` diff
+// eslint-disable-next-line no-unused-vars
+import webpackEntry from 'webpack-entry';
```

This needs to be documented.

**Comment by edmundoa** [comment]
We also need to document how plugins should use `Routes.pluginRoute()` to get links to its pages (and other plugin pages as well).

**Comment by dennisoelkers** [comment]
Fixed in #253.
